dashbuilder-client/dashbuilder-common-client/pom.xml

```diff
@@ -54,6 +54,16 @@
     </dependency>
 
     <dependency>
+      <groupId>org.jboss.errai</groupId>
```

Comment:
I think it's missing to add the GWT modules dependencies for `errai-common` & `errai-ui` into `dashbuider-common-client` module as well